capstone-app/src/app/modules/maps/map/map.component.ts

```diff
@@ -7,5 +7,5 @@ import { Component } from '@angular/core';
 })
 export class MapComponent {
   // these coordinates are for a location in Miami
-  location = new google.maps.LatLng(26.011760, -80.139050);
+  //location = new google.maps.LatLng(26.011760, -80.139050);
```

Comment:
Don't commit or push this file as we don't want the master branch to have this commented out. I fixed the errors it produced in my PR